packages/@react-spectrum/menu/stories/MenuTrigger.stories.tsx

```diff
@@ -712,3 +715,38 @@ function ControlledOpeningMenuTrigger() {
     </div>
   );
 }
+
+export let MenuItemUnavailable = {
```

Comment:
Can we add a dynamic version of this story? I imagine it should work just fine since MenuDialogTrigger's getCollectionNode doesn't need any `renderer` support, but would be good just as an example of what the data + the Menu's render function would look like.